src/frontend/utils/telemetry/Instrumentation.js

```diff
@@ -13,39 +13,33 @@
 // limitations under the License.
 
 const opentelemetry = require('@opentelemetry/sdk-node');
-const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
-const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-grpc');
-const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-grpc');
-const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics');
-const { alibabaCloudEcsDetector } = require('@opentelemetry/resource-detector-alibaba-cloud');
-const { awsEc2Detector, awsEksDetector } = require('@opentelemetry/resource-detector-aws');
-const { containerDetector } = require('@opentelemetry/resource-detector-container');
-const { gcpDetector } = require('@opentelemetry/resource-detector-gcp');
-const { envDetector, hostDetector, osDetector, processDetector } = require('@opentelemetry/resources');
+const {getNodeAutoInstrumentations} = require('@opentelemetry/auto-instrumentations-node');
```

Comment:
Minor, but is there a reason to remove these spaces? Other files do have them, so this will now be inconsistent.

Reply:
this was IntelliJ reformatting the file. I'll configure it to respect the spaces.